File: apiEZPZ/src/main/java/com/example/apiezpz/item/dto/ItemSearchResponse.java
package com.example.apiezpz.item.dto;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Builder
public class ItemSearchResponse {
    private boolean isAllowed;
    private boolean isConditional;
    private String originalText;
    private String restrictions;
    private String name;
    private String category;
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/item/dto/ItemSearchResponse.kt
package com.example.apiezpz.item.dto

data class ItemSearchResponse(
    var isAllowed: Boolean = false,
    var isConditional: Boolean = false,
    var originalText: String = "",
    var restrictions: String = "",
    var name: String = "",
    var category: String = ""
) 